Source/utils/display.h

```diff
@@ -31,6 +31,17 @@ void SetVideoModeToPrimary(bool fullscreen, int width, int height);
 
 bool IsFullScreen();
 
+#ifdef __3DS__

```

Comment:
My comment was about moving this to a platform-specific file, e.g. Source/platform/**ctr**/display.hpp

Reply:
I think I got it right this time.